features/db-check.feature

```diff
@@ -97,3 +97,18 @@ Feature: Check the database
       Access denied
       """
     And STDOUT should be empty
+
+  Scenario: Ensure MySQL defaults are available when as appropriate with --defaults flag
```

Comment:
All of the tests are in one way or the other to "ensure" something, so that term is superfluous here.

```suggestion
  Scenario: MySQL defaults are available as appropriate with --defaults flag
```